### DISCUSSION TITLE
allow self-spectating for tournament client

### DISCUSSION THREAD

**Discussion by ana-rchy** [ppy/osu#25295] [Ideas] [answered]
currently, theres no way for the same person to stream and play in a tournament match, as the tournament client simply doesnt recognize yourself in a lobby, and multiaccounting isnt allowed

this became a problem when hosting/streaming a university knockout tournament that i wanted to also participate in, and there wasnt anyone else who wouldve both had access to the shared twitch channel's stream key, and had an account with supporter

the only solution i can think of would be to add support for self-spectating in stable's tournament client

**Comment by peppy** [comment, marked as answer]
I believe this is already possible in lazer. It won't be implemented in stable though. High effort.

**Reply by ana-rchy** [reply]
so just gotta wait until lazer tournament client happens?
